third_party/move/move-vm/runtime/src/lib.rs

```diff
@@ -46,4 +45,5 @@ pub use storage::{
     },
     module_storage::{ambassador_impl_ModuleStorage, ModuleBytesStorage, ModuleStorage},
     publishing::TemporaryModuleStorage,
+    use_loader_v1_based_on_env,
```

Comment:
maybe it's worth defining it here?

Reply:
Good point! We should remove it as we land, this is mostly to avoid recompilations when running V1 and V2 tests/benchmarks.

Reply:
done